go-samson-secret-puller/secrets.go

```diff
@@ -0,0 +1,36 @@
+package samsonsecretpuller
+
+import (
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+)
+
+// Secrets returns a map of secret names and values
+func Secrets() (map[string]string, error) {
+	s, err := readSecrets("/secrets")
+	return s, err
+}
+
+func readSecrets(d string) (map[string]string, error) {
```

Comment:
can you do optional argument instead ?
`Secrets(d string || "/secrets")` <- not actual go ;)